### PR TITLE
[action] add standalone mode in grant rewarding

### PR DESCRIPTION
**Pull request by huof6829**
# Description

 make run 
./ioctl  node reward pool
the output's example:
```
  Total Unclaimed:	       2560 IOTX
  Total Available:	       199997440 IOTX
  Total Balance:	       200000000 IOTX
```

 ./ioctl  node reward unclaimed  #ioAddr
the output's example:
  `io15x97rtmxnuj0n5meathfqh49hnx258nzey7uzq:  272 IOTX`

Fixes https://github.com/iotexproject/iotex-core/issues/1759

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Code refactor or improvement
- [] Breaking change (fix or feature that would cause a new or changed behavior of existing functionality)
- [] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [] make test
- [] fullsync
- [] Other test (please specify)

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: action/protocol/rewarding/reward.go [hunk 1]
```diff
@@ -94,8 +94,12 @@ func (p *Protocol) GrantBlockReward(
 	}
 	// If reward address doesn't exist, do nothing
 	if rewardAddrStr == "" {
-		log.S().Debugf("Producer %s doesn't have a reward address", producerAddrStr)
-		return nil, nil
+		if p.scheme == _standaloneScheme {
+			rewardAddrStr = producerAddrStr
```
Comment (CoderZhi) [inline action/protocol/rewarding/reward.go:98]: this is an `else` for the `if` in line 83, that `pp` is nil. Or an easier change is to have a poll protocol with only one candidate, who is the producer.

File: api/coreservice.go [hunk 2]
```diff
@@ -156,6 +156,7 @@ type (
 		gs                *gasstation.GasStation
 		broadcastHandler  BroadcastOutbound
 		cfg               config.API
+		scheme            string
```
Comment (CoderZhi) [inline api/coreservice.go:159]: this is weird that we need such a `scheme` in API. we need a better way to read state of rewarding.

**Comment by huof6829** [comment]
My test output:
 
截屏2022-09-08 19 48 36

**Review by huangzhiran — APPROVED** [review]

**Comment by huangzhiran** [comment]
plz fix the code smell, thx

**Comment by dustinxie** [comment]
i think #1759 wants to disable grant reward in standalone mode: 
"reward protocol which queries ProductivityByEpoch cannot return appropriate values in standalone mode"

**Comment by huof6829** [comment]
ok. Node reward is disabled now in standalone mode. No need to correct.

**Pull request closed without merging**